application/app/views/shared/_project_progress.html.erb

```diff
@@ -0,0 +1,66 @@
+<%# Badge counts (ensure values default to 0) %>
+<% completed ||= 0 %>
+<% pending ||= 0 %>
+<% cancelled ||= 0 %>
+<% error ||= 0 %>
+<% total ||= 0 %>
+
+<%
+  if total == 0
+    percent = 0
+  else
+    percent = ((completed + cancelled + error) * 100 / total.to_f).round(1).to_i
+  end
+%>
+<% offset = (114 - (114 * percent.to_f / 100)).round(1).to_i %>
+
+<%# Default colors if not passed %>
+<% progress_color ||= "#0d6efd" %>
+<% background_color ||= "#e9ecef" %>
+<% text_color ||= "#212529" %>
+
+<div id="progress-circle-<%= id %>"
+     class="d-flex align-items-center gap-3"
+     style="width: 100%; max-width: 240px; height: 100px;"><!-- ✅ Unified sizing -->
```

Comment:
I think it is better to use a CSS class instead of inline styles

Reply:
fixed